core/src/main/java/bisq/core/offer/OfferUtil.java

```diff
@@ -488,6 +488,14 @@ public OfferPayload getMergedOfferPayload(OpenOffer openOffer,
         }
     }
 
+    public static boolean isFiatOffer(Offer offer) {
+        return offer.getBaseCurrencyCode().equals("BTC") && !offer.isBsqSwapOffer();
+    }
+
+    public static boolean isAltcoinOffer(Offer offer) {
+        return offer.getCounterCurrencyCode().equals("BTC") && !offer.isBsqSwapOffer();
+    }
```

Comment:
Is swapOffer not considered an altcoin trade? I guess it makes some sense but also somewhat odd.